okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/Platform.java

```diff
@@ -130,6 +135,29 @@ public void connectSocket(Socket socket, InetSocketAddress address,
     socket.connect(address, connectTimeout);
   }
 
+  public static SSLContext getSslContext() throws NoSuchAlgorithmException {
```

Comment:
Oh, I assumed it was okay to edit this file directly, since it seemed to be edited here in [this PR](https://github.com/grpc/grpc-java/pull/1817) (I don't see PR 1817's changes in [this repo](https://github.com/square/okhttp)).  If you rather I move it, do you have a recommendation for where best to add it -- would the OkHttpChannelBuilder class be okay?